src/layouts/PLayoutResizable/PLayoutResizable.vue

```diff
@@ -19,57 +19,129 @@
 </template>
 
 <script lang="ts" setup>
-  import { computed, ref, useSlots } from 'vue'
+  import { useElementRect } from '@prefecthq/vue-compositions'
+  import { computed, ref, useSlots, watch } from 'vue'
+
+  export type PLayoutResizablePlacement = 'left' | 'right' | 'top' | 'bottom'
```

Comment:
Do we have a use case for all these?

Reply:
Only that I wanted to build a flexible drawer component out of the gate (since it influences the overall design) and this is required for support of the resizable drawer 🤷🏻